tsfc/coffee.py

```diff
@@ -296,7 +299,7 @@ def _expression_scalar(expr, parameters):
     if isnan(expr.value):
         return coffee.Symbol("NAN")
     else:
-        return coffee.Symbol(("%%.%dg" % (PRECISION - 1)) % expr.value)
+        return coffee.Symbol(("%%.%dg" % (parameters.precision - 1)) % expr.value)
```

Comment:
Analogically for `0.23456789` tsfc does `0.23457` while quad and uflacs do `0.234568`.